### ISSUE TITLE
Duplicate CSS

### ISSUE THREAD

**Issue by maya** [uswds/uswds#2361]
via Slack:

Two max widths:

```
.usa-grid,
.usa-grid-full {
 max-width: 1200px;
 margin-left: auto;
 margin-right: auto;
 max-width: 1040px;
}
```

this is occurs twice:

```
html {
 box-sizing: border-box;
}

*,
*::before,
*::after {
 box-sizing: inherit;
}
```

**Comment by stphnwlkr** [comment]
Asked the same question on Slack a year ago (March 12)... Shawn Allen replied "it's mostly an artifact of how we've structured our Sass—specifically, using `@include` vs. `@extend` :confused:
there are tools that you (or we) could use to post-process the CSS and optimize selectors and properties a _lot_ more 

the flip side is that debugging the CSS becomes a _lot_ harder when you optimize it like that."

**Comment by maya** [comment]
@stphnwlkr I assume you're referring to the first issue about grids, not the second issue? 

I think it could be fixed by making better use of the neat `outer-container` mixin it includes:

```
@mixin outer-container($local-max-width: $max-width) {
  @include clearfix;
  max-width: $local-max-width;
  margin: {
    left: auto;
    right: auto;
  }
}
```

However, I don't want to get too deep into it in case we end up upgrading Bourbon + Neat to the latest version or use a different grid system.

**Comment by maya** [comment]
Removing the box-sizing CSS from base.scss fixes the first issue, but I don't know where the box-sizing code is coming from that gets added as the first thing in the CSS file and would like to know.

**Comment by stphnwlkr** [comment]
Yes. Sorry I wasn’t clear on that.

**Comment by maya** [comment]
Closed via #2404.
